google/cloud/spanner/client.h

```diff
@@ -256,6 +256,26 @@ class Client {
                                  SqlStatement statement);
   //@}
 
+  /**
+   * Executes a Partitioned DML SQL query.
+   *
+   * @param statement the SQL statement to execute. Please see the
+   *     [spanner documentation][dml-partitioned] for the restrictions on the
+   *     SQL statements supported by this function.
+   *
+   * @par Example
+   * @snippet samples.cc execute-sql-partitioned
+   *
+   * @see [Partitioned DML Transactions][txn-partitioned] for an overview of
```

Comment:
Do we need to mention "Partitioned DML Transactions" to the user now?

Reply:
Removed.